c_src/membrane_audio_mix_plugin/caps_audio_raw.c

```diff
@@ -0,0 +1,102 @@
+#include "caps_audio_raw.h"
```

Comment:
Simply adding to `raw_audio_format` is a no-go, we don't want to add native dependencies in `format` definition - it would either have to be done clever enough to not include the native part when it's not needed or placed in a different spot. And I belive is much better candidate for that